File: mt9x9.rs
// 9x9 multiplication table in Rust
// CC0, Wei-Lun Chao <bluebat@member.fsf.org>, 2018.
// rustc mt9x9.rs ; ./mt9x9

fn main() {
    for i in (1..10).step_by(3) {
        for j in 1..10 {
            for k in i..i+3 {
                print!("{}x{}={:2}\t", k, j, k*j);
            }
            println!();
        }
        println!();
    }
}

File: mt9x9.pas
{ 9x9 multiplication table in Pascal }
{ CC0, Wei-Lun Chao <bluebat@member.fsf.org>, 2018. }
{ fpc mt9x9.pas ; ./mt9x9 }

program mt9x9;
uses SysUtils;
var
    i, j, k : Integer;
begin
    for i in [1, 4, 7] do
    begin
        for j := 1 to 9 do
        begin
            for k := i to i+2 do
            begin
                Write(Format('%dx%d=%2d'#9, [k, j, k*j]));
            end;
            Write(#10);
        end;
        Writeln;
    end;
end.
